crates/pop-drink/README.md

```diff
@@ -0,0 +1,60 @@
+# Pop Sandbox
```

Comment:
This document need to be updated I guess.

Reply:
This entire README need to be reworked but lets do that in a separate PR

Reply:
Created issue